language/sygus.tex

```diff
@@ -2229,6 +2289,30 @@ \subsection{Satisfying Semantic Specifications}
 which is why their definitions are explicitly given in the definition of
 $\Psi_{\alpha}$.
 
+\subsubsection{Satisfying Semantic Specifications for Optimization Queries}
+We extend the notion of semantic correctness to responses to optimization
+queries of the form:
+\[
+\paren{\optsynthkwd\mbox{ }\paren{t_1 \ldots t_j}\mbox{ } A_1 \ldots A_k}
+\]
+Recall from \cref{sec:output} that a successful response to this command
+contains:
+\begin{enumerate}
+\item A tuple of terms $(c_1, \ldots, c_j)$ corresponding to the valuation of terms $(t_1, \ldots, t_j)$, and
+\item A tuple of functions $( \lambda X_1 \ldotp t_1, \ldots, \lambda X_n \ldotp t_n)$
```

Comment:
We have also overloaded $t_1$ here.

May be use $t'_i$ in place of $t_i$ within `optimize-synth`?